catalog.html

```diff
@@ -0,0 +1,274 @@
+                                </label>
+                            </li>
+                            <li class="catalog-filter-item">
+                                <label class="control">
+                                    <input class="control-item" type="checkbox" name="parking" checked>
+                                    Парковка
+                                </label>
+                            </li>
+                            <li class="catalog-filter-item">
+                                <label class="control">
+                                    <input class="control-item" type="checkbox" name="wifi">
+                                    Wi-Fi
+                                </label>
+                            </li>
+                        </ul>
+                    </fieldset>
+                    <fieldset class="catalog-filter-group">
+                        <legend class="catalog-filter-title">Тип жилья</legend>
+                        <ul class="catalog-filter-list">
+                            <li class="catalog-filter-item">
+                                <label class="control">
+                                    <input class="control-item" type="radio" name="housing-group" value="hotel" checked>
+                                    Гостиница
+                                </label>
+                            </li>    
+                            <li class="catalog-filter-item">
+                                <label class="control">
+                                    <input class="control-item" type="radio" name="housing-group" value="motel">
+                                    Мотель
+                                </label>
+                            </li>
+                            <li class="catalog-filter-item">
+                                <label class="control">
+                                    <input class="control-item" type="radio" name="housing-group" value="apartments">
+                                    Апартаменты
+                                </label>
+                            </li>
+                        </ul>
+                    </fieldset>
+                    <fieldset class="catalog-filter-group">
+                        <legend class="catalog-filter-title">Стоимость, ₽:</legend>
+                        <p class="cost-inputs">
+                            <label class="cost-label">от
+                                <input class="cost-input" type="number" min="0" max="12000" value="0">
+                            </label> 
+                            <label class="cost-label">до
+                                <input class="cost-input" type="number" min="0" max="12000" value="9000">
+                            </label>
+                        </p>                        
+                        <input class="range-slider-input" type="range" min="0" max="12000" value="0" step="100" aria-label="minrange">
```

Comment:
`aria-label` будет читаться скринридером, он должен быть понятным
```suggestion
                        <input class="range-slider-input" type="range" min="0" max="12000" value="0" step="100" aria-label="Ползунок минимальной цены">
```